app/components/markdown/at_mention/at_mention.tsx

```diff
@@ -142,7 +142,9 @@ const AtMention = ({
     // Effects
     useEffect(() => {
         // Fetches and updates the local db store with the mention
-        fetchUserOrGroupsByMentionsInBatch(serverUrl, mentionName);
+        if (!user.username) {
```

Comment:
This was getting an issue with mentions with trailing punctuation marks. May not be the best solution, though.

Reply:
Yes. There might be still some problem if we have something like a user `user1` and another user `user1.`, and we only have the first one on the database, but I think that would be not common.